website/docs/pubsub.md

```diff
@@ -43,185 +43,191 @@ data:
 #### Available Pubsub drivers
 Dapr: https://dapr.io/
 
-### Violations
-
-The audit pod publishes violations in following format:
-
-```json
-{
-  "id": "2023-07-18T21:21:52Z",
-  "details": {
-    "missing_labels": [
-      "test"
-    ]
-  },
-  "eventType": "violation_audited",
-  "group": "constraints.gatekeeper.sh",
-  "version": "v1beta1",
-  "kind": "K8sRequiredLabels",
-  "name": "pod-must-have-test",
-  "message": "you must provide labels: {\"test\"}",
-  "enforcementAction": "deny",
-  "resourceAPIVersion": "v1",
-  "resourceKind": "Pod",
-  "resourceNamespace": "nginx",
-  "resourceName": "nginx-deployment-cd55c47f5-2b84x",
-  "resourceLabels": {
-    "app": "nginx",
-    "pod-template-hash": "cd55c47f5"
-  }
-}
-```
-
 ### Quick start with publishing violations using Dapr and Redis
 
-> Redis is used for example purposes only. Dapr supports [many different state store options](https://docs.dapr.io/reference/components-reference/supported-state-stores/).
-
 #### Prerequisites
 
 1. Install Dapr
 
-    To install Dapr with specific requirements and configuration, please refer to [Dapr docs](https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/).
-    > Dapr is installed with mtls enabled by default, for more details on the same plaase refer to [Dapr security](https://docs.dapr.io/operations/security/mtls/#setting-up-mtls-with-the-configuration-resource).
+   To install Dapr with specific requirements and configuration, please refer to [Dapr docs](https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/).
+    > Make sure to set `SIDECAR_DROP_ALL_CAPABILITIES` environment variable on `dapr-sidecar` injector pod to `true` to avoid getting `PodSecurity violation` errors for the injected sidecar container as Gatekeeper by default requires workloads to run with [restricted](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted) policy. If using helm charts to install Dapr, you can use `--set dapr_sidecar_injector.sidecarDropALLCapabilities=true`. Additionally, [configure appropriate seccompProfile for sidecar containers](https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-production/#configure-seccompprofile-for-sidecar-containers) injected by Dapr to avoid getting `PodSecurity violation` errors.
```

Comment:
re: seccomp, looks like there's `global.seccompProfile`. can we use that?
https://github.com/dapr/dapr/blob/master/charts/dapr/README.md

Reply:
can we add bullet points for each so it's more readable?

Reply:
`global.seccomProfile` relates to the security profile of dapr pods (dapr-operator, dapr-sidecar-injector, dapr-placement pods), we need dapr annotation on audit pod for `seccompProfile` that tells `sidecar-injector` of dapr about what `seccomProfile` to set on the `sidecar` that is to be injected in audit pod.